projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -199,22 +450,24 @@ export function addCommentAndWait(text: string) {
 }
 
 /**
- * checks whether the given header comment is displayed on the given position
- * @param index position of the comment, starting with 0 for the first comment.
- * @param text text to be displayed
+ * Checks whether the given header comment is displayed on the given position
+ * @param index Position of the comment, starting with 0 for the first comment.
+ * @param text Text to be displayed
  */
 export function checkComment(index: number, text: string) {
+  log(checkComment.name);
```

Comment:
I would suggest not to log the name of the method but to log the method description, for instance:
> Verifies a comment

Reply:
done